mvp-spec/partials-updater/src/interpretor.ts

```diff
@@ -57,87 +61,67 @@ export async function interpret(tokens: LexerToken[]): Promise<string> {
     return doc;
 }
 
-async function interpretPartialBeginToken(token: LexerToken) : Promise<string> {
-    const jsonStr = token.text
-                    .slice(PartialBeginStartKey.length)
-                    .slice(0, -PartialBeginEndKey.length);
+async function interpretPartialBeginToken(token: LexerToken, lineBreak: string): Promise<string> {
+    const jsonStr = token.text.match(PartialBegin)[1];
     try {
         const config: PartialConfig = JSON.parse(jsonStr);
-        if (!config.hasOwnProperty('files') && config.files.length >= 1) {
-            throw new Error(`Partial with bad configuration: ${token.text}`)
+        if (config.files?.length < 1) {
+            throw ''
```

Comment:
Can we throw something cleaner here ?

Reply:
No, because this exception is caught just below, by the catch.
I will rework this block anyway because below there is a nested exception as well.